scripts/init-properties-for-docker.sh

```diff
@@ -17,11 +17,11 @@ if [ -f $resourcefile ]; then
 fi
 
 cat > $resourcefile <<EOF
-ctia.store.es.default.host=$ip
+ctia.store.es.default.host="$ip"
```

Comment:
I did not see the:
`cat > $resourcefile <<EOF`
in the previous context, so I did not realize it was going to a properties file, sorry. In that case you do not want quotes. But the move to host/port is preferable to URI.